src/main/java/uk/gov/hmcts/darts/retention/controller/RetentionController.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.darts.retention.controller;
```

Comment:
@davet1985 Should this be in a package called `retentions`, or is it ok?

Reply:
The majority of them are singular, so I'd say this is fine.